zer0-os/zOS
# feat: implement removing favorite room tags

### What does this do?
- removes room from favorites (matrix)
- handles room unfavorited updates (saga)
- adds and handles action for unfavoriting room
- updates the publishing of room tag updates

### Why are we making this change?
- to be able to mark rooms (conversations) NOT as favorites (unfavoriting) for an enhanced user experience

### How do I test this?
- run tests as usual.
- alternatively, you could call the `removeRoomFromFavorites` and check the tag event is logged and the event content tags no longer contain the `m.favorite` tag.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


REMOVE Favorite Demo (once wired up) with rough three dot menu

https://github.com/zer0-os/zOS/assets/39112648/98819d9c-f182-4ca1-aa96-852e45d874c9
